tests/core/editable/keystrokes/delbackspacequirks/manual/nbspdeleting.html

```diff
@@ -0,0 +1,11 @@
+<textarea id="editor" cols="30" rows="10">
+	<p>Hello World!</p>
+</textarea>
+
```

Comment:
In this test let's add a paragraph below the editor tracking the editor source code on change event to improve debugging experience too.